src/metabase/sync/analyze/fingerprint/datetime.clj

```diff
@@ -0,0 +1,17 @@
+(ns metabase.sync.analyze.fingerprint.datetime
+  "Logic for generating a `DateTimeFingerprint` from a sequence of values for a `:type/DateTime` Field."
+  (:require [metabase.sync.interface :as i]
+            [schema.core :as s]))
+
+(s/defn datetime-fingerprint :- i/DateTimeFingerprint
+  "Generate a fingerprint containing information about values that belong to a `DateTime` Field."
+  [values :- i/FieldSample]
+  (let [[value & values] values]
+    (zipmap [:earliest :latest]
```

Comment:
is this really the simplest way to get a min/max or a set of values?

Reply:
or are we doing this for performance reasons?

Reply:
Maybe to avoid two passes over the data. Are there going to be a huge amount of dates here?